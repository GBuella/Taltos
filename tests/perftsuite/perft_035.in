setboard r3k1r1/8/8/8/8/8/8/R3K2R b KQq - 0 1
echo r3k1r1/8/8/8/8/8/8/R3K2R b KQq - 0 1
qperft 1
qperft 2
qperft 3
qperft 4
qperft 5
qperft 6
