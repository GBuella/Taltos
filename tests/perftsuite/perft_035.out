r3k1r1/8/8/8/8/8/8/R3K2R b KQq - 0 1
25
547
13579
316214
7878456
189224276
